ArduPlane/servos.cpp

```diff
@@ -890,7 +890,11 @@ void Plane::set_servos(void)
     // slew rate limit throttle
     throttle_slew_limit(SRV_Channel::k_throttle);
 
+    int8_t min_throttle = 0;
 #if AP_ICENGINE_ENABLED
+    if (g2.ice_control.allow_throttle_while_disarmed()) {
+        min_throttle = MAX(aparm.throttle_min.get(), 0);
+    }
```

Comment:
I think this could still be down in the switch statement?

Reply:
could, but then we have 3 ugly ifdefs .... I think its nicer with 2